docs/index.rst

```diff
@@ -1,14 +1,162 @@
-Welcome to Betelgeuse's documentation!
-======================================
+Betelgeuse documentation
+========================
 
-Betelgeuse reads standard Python test cases and offers tools to interact with
-Polarion. Possible interactions are:
+.. contents:: Topics
 
-* Automatic creation and update of Requirements and Test Cases from a Python
+.. _what_is_betelgeuse:
+
+What is Betelgeuse?
+```````````````````
+
+Betelgeuse is a python program that reads standard Python test cases and offers
+tools to interact with Polarion. Possible interactions are:
+
+* Automatic creation/updation of Requirements and Test Cases from a Python
   project code base.
 * Creation of Test Runs based on a jUnit XML file.
 
-Documentation contents:
+Betelgeuse uses `testimony <https://pypi.python.org/pypi/testimony>`_ and
+pylarion projects to parse test cases and communicate with polarion
+respectively.
+
+.. _prerequisites:
+
+Prerequisites
+`````````````
+Login to polarion in a browser to check if the login user has permissions to do
+the following:
+
+* add/update test case
```

Comment:
ok I think you mean `make it consistent`.  Please check the updated commit and let me know what do you think.